client/my-sites/themes/current-theme/index.jsx

```diff
@@ -90,7 +87,7 @@ const CurrentThemeWithOptions = ( { site, currentTheme } ) => (
 );
 
 export default connect(
-	( state, { site } ) => ( {
-		currentTheme: site && getCurrentTheme( state, site.ID )
+	( state, { siteId } ) => ( {
+		currentTheme: siteId && getCurrentTheme( state, siteId )
```

Comment:
Minor note -- I think we can safely remove the `siteId &&` check here [because of the way `getCurrentTheme` is implemented](https://github.com/Automattic/wp-calypso/blob/08189683ebb04c9d3ab5dfbea068b322a76453bd/client/state/themes/current-theme/selectors.js#L4).